site/docs/use-cases/image-processing/_sections/_run_model/index.mdx

```diff
@@ -1,10 +1,12 @@
 import CodeExampleCPP from './_code_example_cpp.mdx';
 import CodeExamplePython from './_code_example_python.mdx';
 
-## Run Model Using OpenVINO™ GenAI
+## Run Model Using OpenVINO GenAI
 
-OpenVINO GenAI introduces `Text2ImagePipeline` for inference of text-to-image models such as Stable Diffusion 1.5, 2.1, XL, LCM, Flux, and more.
-See all supported [image generation models](/docs/supported-models/#image-generation-models).
+OpenVINO GenAI introduces the `VLMPipeline` pipeline for inference of multimodal text-generation Vision Language Models (VLMs).
+With a text prompt and an image as input, `VLMPipeline` can generate text using models such as LLava or MiniCPM-V.
```

Comment:
```suggestion
With a text prompt and an image as input, `VLMPipeline` can generate text using models such as LLaVA or MiniCPM-V.
```
Putting links to the models behind LLaVA or MiniCPM-V would be handy, can you add them

Reply:
As we already have link to VLM supported models, I removed mentioning of LLaVA and MiniCPM-V here.